include/notifier.php

```diff
@@ -184,18 +185,18 @@ function notifier_run(&$argv, &$argc){
 			intval($parent_id)
 		);
 
-		if(! count($items)) {
+		if (! count($items)) {
 			return;
 		}
 
 		// avoid race condition with deleting entries
 
-		if($items[0]['deleted']) {
-			foreach($items as $item)
+		if ($items[0]['deleted']) {
+			foreach ($items as $item)
```

Comment:
Sorry I didn't catch that one, but braces seem missing from this `foreach` statement.